<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AP_Widget</class>
 <widget class="QWidget" name="AP_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>469</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>AP_Widget</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>221</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="AP_InfoLayout">
    <item>
     <widget class="QLabel" name="AP_NameLabel">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(242, 184, 12);color:rgb(255,255,255)</string>
      </property>
      <property name="text">
       <string>歌曲：未知</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="AP_AlbumLabel">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(242, 184, 12);color:rgb(255,255,255)</string>
      </property>
      <property name="text">
       <string>专辑：未知</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="AP_SingerLabel">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(242, 184, 12);color:rgb(255,255,255)</string>
      </property>
      <property name="text">
       <string>歌手：未知</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="AP_ControlLayout">
    <item>
     <widget class="QPushButton" name="AP_PreOne">
      <property name="text">
       <string>上一首</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="AP_PLay">
      <property name="text">
       <string>播放/暂停</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="AP_NextOne">
      <property name="text">
       <string>下一首</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>591</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="AP_TimeControlLayout">
    <item>
     <widget class="QSlider" name="AP_TimeBar">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="AP_TimeLabel">
      <property name="text">
       <string>0:00/0:00</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="AP_CoverLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>220</width>
     <height>220</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>420</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="AP_VolumeLayout">
    <item>
     <widget class="QPushButton" name="AP_Silence">
      <property name="text">
       <string>静音</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="AP_VolumeBar">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="AP_VolumeLabel">
      <property name="text">
       <string>50</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>380</y>
     <width>171</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="AP_ModeLayout">
    <item>
     <widget class="QLabel" name="AP_ModeLabel">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(242, 184, 12);color:rgb(255,255,255)</string>
      </property>
      <property name="text">
       <string>当前模式（点击切换）</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="AP_ListMode">
      <property name="text">
       <string>列表循环</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>261</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="AP_ListLayout">
    <item>
     <layout class="QHBoxLayout" name="AP_ListButtonLayout">
      <item>
       <widget class="QPushButton" name="AP_Select">
        <property name="text">
         <string>新建列表</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="AP_Destory">
        <property name="text">
         <string>清除列表</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="AP_Insert">
        <property name="text">
         <string>插入歌曲</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="AP_ListNotion">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(242, 184, 12);color:rgb(255,255,255)</string>
      </property>
      <property name="text">
       <string>列表中没有歌曲</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="AP_List"/>
    </item>
    <item>
     <widget class="QLabel" name="AP_ListOp">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(242, 184, 12);color:rgb(255,255,255)</string>
      </property>
      <property name="text">
       <string>单击曲目跳转播放             双击曲目删除</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>259</x>
     <y>10</y>
     <width>251</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="AP_LyricLayout">
    <item>
     <widget class="QLabel" name="AP_Lyric1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="AP_Lyric2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="AP_Lyric3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="AP_Lyric4">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="AP_Lyric5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="AP_Lyric6">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="AP_Lyric7">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
